### ISSUE TITLE
Initialization only on a small part of the image

### ISSUE THREAD

**Issue by laxnpander** [yunchih/ORB-SLAM2-GPU2016-final#11]
Hey guys,

nice work you have done here! I was able to speed up my orb slam by factor of 4. However, for initialization somehow only a small part of the image is used for matching (see attachment). As soon as the orb slam is initialized, features are tracked over the image as a whole, but before that only that small area is used. This gets even worse, if I use bigger images.
 Any suggestions which part of the code might do this?

![selection_041](https://user-images.githubusercontent.com/20966574/36039178-096d373e-0dc2-11e8-9b45-948f656c6611.png)

Thanks and best regards,
Alex

**Comment by yunchih** [comment]
Hi, do you encounter the same issue in the pure CPU version?  :D

**Comment by laxnpander** [comment]
@yunchih Nope, the cpu version works fine. For which OpenCV version was that developed? I was using 3.3.1

**Comment by yunchih** [comment]
Our development was based on OpenCV 3, too, back in the day when the upstream project was still based on OpenCV 2.  However, there hasn't been porting being done yet, and I've been busy in other work.  It'd be good if you got patches to submit, which would likely benefit a lot of people.
